lib/repl.js

```diff
@@ -794,7 +794,16 @@ function complete(line, callback) {
     filter = match[1];
     var dir, files, f, name, base, ext, abs, subfiles, s;
     group = [];
-    var paths = module.paths.concat(require('module').globalPaths);
+    let paths = [];
+    if (completeOn === '.') {
+      group = ['./', '../'];
```

Comment:
1) All directories in the current working directory will be autocompleted if user types `./` instead of just `.`. AFAICT, that's how bash works too, it just completes `.` to `./` first unconditionally.

2) Right, I forgot about them. Thanks.